todo-with-kuzzle-and-vueJS/src/main.js

```diff
@@ -0,0 +1,45 @@
+import Vue from 'vue'
+import App from './App.vue'
+import router from './router'
+import store from './store'
+
+/* eslint-disable */
+
+
+Vue.config.productionTip = false
+
+new Vue({
+  router,
+  store,
+  render: h => h(App)
+}).$mount('#app')
+
+
+const {
```

Comment:
I think a good practice would be to have a separate file with the instantiation of kuzzle and network listener, maybe in `service/kuzzle.js`